index.html

```diff
@@ -50,5 +51,199 @@ <h1 class="headline-header">I'm Ismail Glade to see you!</h1>
         </ul>
       </div>
     </section>
+
+    <!-- the Works section -->
+
+    <section class="works-section">
+      <div class="work-card work-cardFirst">
```

Comment:
- [ ] **Please use `<ul>` to group the project and `li` element to display each project**

What this means is that you should use `ul` to group all the project together in one element. Then, inside of that `ul` element, you should use `li` to display each project and it's contents. 

Why is it important? 

Whenever you are trying to group and list things, you should use `li` tags. HTML has the ability to express lists of things, and it helps the Google robot, screen readers, and all manner of users who don't care solely about the presentation of the site understand your content better.

Reply:
This has been solved 👍